bitnami/postgresql/values.yaml

```diff
@@ -95,6 +95,16 @@ serviceAccount:
   ## Name of an already existing service account. Setting this value disables the automatic service account creation.
   # name:
 
+## Pod Security Policy
+## ref: https://kubernetes.io/docs/concepts/policy/pod-security-policy/
+psp:
+  create: false
+
+## Creates role for ServiceAccount
+## Required for PSP
+rbac:
+  create: false
```

Comment:
We should include a validation (see `{{- define "postgresql.validateValues" -}}` macro in the `_helpers.tpl`) to shown an error when `psp.create=true` but `rbac.create=false`

Reply:
@juan131 Thanks for suggestion! Added validation 👍